nvflare/apis/utils/fl_context_utils.py

```diff
@@ -39,6 +39,9 @@ def get_serializable_data(fl_ctx: FLContext):
 
 
 def generate_log_message(fl_ctx: FLContext, msg: str):
```

Comment:
should we make fl_ctx Optional[FLContext]?

Reply:
Not related to this PR. Let's do it in a separate PR (assigned to Yuan-Ting).  :-)